BraintreeCore/src/main/java/com/braintreepayments/api/core/MerchantRepository.kt

```diff
@@ -0,0 +1,19 @@
+package com.braintreepayments.api.core
+
+import android.content.Context
+import android.net.Uri
+import androidx.annotation.RestrictTo
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class MerchantRepository {
```

Comment:
Not sure if there's a clearer name for this class.

Reply:
Maybe a doc string for our internal understanding of what this class is for?

Reply:
➕ 1 I think some docstrings would help, I don't have any issues with the name since this naming seems like a common pattern on Android.

Reply:
does the `@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)` prevent dokka from publishing to the generated docs?

Reply:
It looks like those internal classes do get published :(

https://braintree.github.io/braintree_android/BraintreeCore/com.braintreepayments.api.core/index.html

I'll look to see if there's a way to prevent this.

Reply:
> does the `@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)` prevent dokka from publishing to the generated docs?

Not sure if it does that, but you could add `@suppress` in the docs to suppress it.
Also see: https://github.com/Kotlin/dokka/issues/2404 for a custom implementation we can do to suppress all `@RestrictTo` annotated methods.

Reply:
Dang, I thought there'd be support directly in Dokka for that. I've added some docs for this class.

I'll create a tech debt ticket for possibly adding a custom implementation to suppress based on the annotation.